scalding-beam/src/test/scala/com/twitter/scalding/beam_backend/BeamBackendTests.scala

```diff
@@ -389,6 +389,31 @@ class BeamBackendTests extends FunSuite with BeforeAndAfter {
     )
   }
 
+  test("Force to Disk execution") {
+    val bmode = BeamMode.default(pipelineOptions)
+    val tmpPath1 = tmpPath("tp1")
+    val tmpPath2 = tmpPath("tp2")
+
+    val forcedExecution =
+      TypedPipe
+        .from(Seq(5, 3, 2, 6, 1, 4))
+        .forceToDiskExecution
+
+    val tp1 = forcedExecution.flatMap { forced =>
+      forced.filter(_ % 2 == 0).map(_.toString).writeExecution(TextLine(tmpPath1))
+    }
+
+    val tp2 = forcedExecution.flatMap { forced =>
+      forced.filter(_ % 2 == 1).map(_.toString).writeExecution(TextLine(tmpPath2))
+    }
+
+    Execution.zip(tp1, tp2).waitFor(Config.empty, bmode)
```

Comment:
would be good to test somehow that the forceToDisk data has been cleaned by this point.